examples/postprocess_dag_test_srst2_single_read.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Please remove this file also. If you want a single-read example for AMR, make a standalone example like amr_dag_single.json.

Reply:
resolved